src/MainWindow.cpp

```diff
@@ -359,6 +359,20 @@ MainWindow::event(QEvent *event)
     return QQuickView::event(event);
 }
 
+// HACK: https://bugreports.qt.io/browse/QTBUG-83972, qtwayland cannot auto hide menu
+void
+MainWindow::mousePressEvent(QMouseEvent *event)
```

Comment:
Since this only works on the MainWindow, it won't work on any child windows (like when opening a window separately). I think that is fine though. The upstream bug fix will hopefully fix this anyway. Probably better than adding this to EVERY window.